docs/apis-tools/frontend-development/03-forms/03-customize-and-extend/01-styling.md

```diff
@@ -0,0 +1,172 @@
+  --color-borders-inverted: var(--cds-border-inverse, var(--color-grey-225-10-90));
+  --color-warning: var(--cds-text-error, var(--color-red-360-100-45));
+  --color-warning-light: var(--cds-text-error, var(--color-red-360-100-92));
+  --color-accent: var(--cds-link-primary, var(--color-blue-205-100-40));
+  --color-accent-readonly: var(
+    --cds-border-strong,
+    var(--cds-border-strong-01, var(--color-grey-225-10-55))
+  );
+  --color-datepicker-focused-day: var(--cds-button-primary, var(--color-grey-225-10-55));
+  --color-shadow: var(--cds-shadow, var(--color-grey-225-10-85));
+ 
+  /* font + text settings */
+  --font-family: "IBM Plex Sans", sans-serif;
+  --font-size-group: 15px;
+  --font-size-base: 14px;
+  --font-size-input: 14px;
+  --font-size-label: 12px;
+  --line-height-base: 20px;
+  --line-height-input: 18px;
+  --line-height-label: 16px;
+  --letter-spacing-base: .16px;
+  --letter-spacing-input: .16px;
+  --letter-spacing-label: .32px;
+
+  /* field settings */
+  --form-field-height: 36px;
+  --border-definition: 1px solid var(--color-borders);
+  --border-definition-adornment: 1px solid var(--color-borders-adornment);
+  --outline-definition: 1px solid var(--cds-focus, var(--color-borders));
+  --button-warning-outline-definition: 2px solid var(--color-warning);
+  --border-definition-disabled: 1px solid var(--color-borders-disabled);
+  --border-definition-readonly: 1px solid var(--color-borders-readonly);
+}
+```
+
+### Styleable classes
+
+The simplest way to find the right styleable elements to override is inspecting form-js using your browser's developer tools. Scope rules with the `.fjs-container` class to prevent CSS conflicts. 
+
+For example, to override field borders for single-line fields:
+
+```css
+.fjs-container .fjs-input-group {
+    border-width: 0 0 1px 0;
+}
+```
+
+### Example
+
+Camunda 8 web applications are built using the IBM Carbon design system. Forms rendered in Tasklist appear in this design system by default. Visit the [Carbon form-js styles repository](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-carbon-styles) <GHIcon /> to learn how to create your own form styles.
+
+<div style={ { display: 'flex', gap: '8px', flexWrap: 'wrap' } }>
+
+<div style={ { width: '450px' } } >
+<h4>Basic style</h4>
+<FormViewer schema={ exampleForm } />
+</div>
+
+<div style={ { width: '450px' } } >
+<h4>Custom style (Material-like)</h4>
+<FormViewer customClass="materialized" schema={ exampleForm } />
+</div>
+
+</div>
+
+## Styling via custom renderers
+
+The [form viewer](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-viewer) <GHIcon /> contains all [basic form components](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-viewer/src/render/components/form-fields) <GHIcon /> shipped in Camunda Forms. For full flexibility, fork the library and change the returned HTML of the individual components.
```

Comment:
Alternatively, overwrite the entire thing via a custom component (need to verify that's true, but pretty sure you can just do that). Avoids forking the lib. 

Also I think the title might make people believe we have an extension point at the render level (one day 🙏), maybe we can get rid of that ambiguity in the title?

Reply:
Agree!